wordpress-mobile/WordPress-iOS
# Remove feed from followed sites list in setting

Fixes #10639 

This PR adds the possibility to check and store if a _ReaderSiteTopic_ is a feed or not.
I added a new CoreData model version **86** in order to store the _ReaderSiteTopic_ `isFeed` property.
I also updated _WordPressKit_ to get the isFeed value from the remote model.
In the _NotificationSettingsViewController_ I filtered the followed sites removing the feeds. I also fixed a bug on displaying the right number of rows if the array count is less than the threshold.

**To test:**
- Run this branch over develop to test the migration of the new datamodel.
- Go to the _Reader_ > _Followed Sites_ > _Manage_
- Enter **nytimes.com** in the textfield and press _Done_ and wait till the notice appears saying the site has been added.
- Go to _Me_ > _Notifications Settings_. _The New York Times_ site shouldn't appear on the Followed sites list.

Update release notes:


ccing @theck13
